apps/site/assets/ts/stop/__tests__/StopMapRedesignTest.tsx

```diff
@@ -11,19 +12,54 @@ jest.mock("../../hooks/useMapConfig", () => ({
   })
 }));
 
+jest.mock("../../hooks/useVehiclesChannel", () => ({
+  __esModule: true,
+  default: jest.fn(() => [])
+}));
+
 const testStop = {
   id: "Test Stop ID",
+  name: "Test Stop Name",
   latitude: newLatOrLon(),
   longitude: newLatOrLon()
 } as Stop;
 
+const v1 = {
+  id: "y1799",
+  route_id: "39",
+  stop_id: "72",
+  trip_id: "25",
+  shape_id: "shape_1",
+  direction_id: 1 as DirectionId,
+  status: "STOPPED",
+  latitude: 2.2,
+  longitude: 1.1,
+  bearing: 140,
+  crowding: null
+};
+const v2 = {
+  id: "y1800",
+  route_id: "39",
+  stop_id: "73",
+  trip_id: "25",
+  shape_id: "shape_1",
+  direction_id: 1 as DirectionId,
+  status: "STOPPED",
+  latitude: 2.4,
+  longitude: 1.3,
+  bearing: 141,
+  crowding: null
+};
+
 describe("StopMapRedesign", () => {
   it("should render the Map component with a marker", () => {
-    const { getByAltText } = render(
+    const { getByRole } = render(
```

Comment:
nit-pick (non-blocking): Is there a reason this is used instead of the `screen.getByRole()`